website/docs/r/security_tag.html.markdown

```diff
@@ -0,0 +1,51 @@
+---
+layout: "vcd"
+page_title: "VMware Cloud Director: security_tag"
+sidebar_current: "docs-vcd-resource-security-tag"
+description: |-
+Provides a VMware Cloud Director Security Tag resource. This can be
+used to assign security tag to VMs.
+---
+
+# vcd\_security\_tag
+
+Provides a VMware Cloud Director Security Tag resource. This can be
+used to assign security tag to VMs.
+
+Supported in provider *v3.7+* and requires VCD 10.3.0+
+
+-> **Note:** This resource requires either system or org administrator privileges.
+
+## Example Usage
+
+```hcl
+resource "vcd_security_tag" "my_tag" {
+  name   = "test-tag"
+  vm_ids = [vcd_vm.my-vm-one.id, vcd_vm.my-vm-two.id]
+}
+```
+## Argument Reference
+
+The following arguments are supported:
+
+* `org` - (Optional) The name of organization to use, optional if defined at provider level. Useful when connected as sysadmin working across different organisations
+* `name` - (Required) The name of the security tag.
+* `vm_ids` - (Required) List of VM IDs that the security tag is going to be applied to.
+
+-> The ID of `vcd_security_tag` is set to its name since VCD behind the scenes doesn't create an ID.
+
+# Importing
+
+~> **Note:** The current implementation of Terraform import can only import resources into the state.
+It does not generate configuration. [More information.](https://www.terraform.io/docs/import/)
```

Comment:
Please return comments about `.` usage as now org.securitytag is used :)

Reply:
Done!